store/db.js

```diff
@@ -0,0 +1,5 @@
+var level = require('level')
+
+module.exports = function (state, emitter) {
+  state.db = level('peermaps-web')
```

Comment:
This should be initialized first and other states can access `state.db` and create their own `subleveldown` instances (just like settings does)